scripts/coverage.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+export SOLIDITY_COVERAGE=true
+./node_modules/.bin/solidity-coverage
+export SOLIDITY_COVERAGE=false
```

Comment:
`SOLIDITY_COVERAGE=true ./node_modules/.bin/solidity-coverage` should be equivalent to these 3 lines